azurerm/internal/services/mixedreality/spatial_anchors_account_resource.go

```diff
@@ -126,6 +136,8 @@ func resourceSpatialAnchorsAccountRead(d *schema.ResourceData, meta interface{})
 	if location := resp.Location; location != nil {
 		d.Set("location", azure.NormalizeLocation(*location))
 	}
+	d.Set("account_domain", resp.AccountDomain)
```

Comment:
Persumably we should nil check for `resp.Properties` here?

Reply:
This wants to be:

```go
if props := resp.SpatialAnchorsAccountProperties; props != nil {
  d.Set("account_domain", props.AccountDomain)
  d.Set("account_id", props.AccountID)
}
```

The reason is that the API might return response with `SpatialAnchorsAccountProperties` being absent, which will cause the dereference shortcut panic.

Reply:
aaah, makes sense now. Fixing, thanks!